exercises/codec-server/README.md

```diff
@@ -0,0 +1,119 @@
+
+During this exercise, you will:
+
+* Review a Codec Server implementation
+* Configure a Codec Server to share your custom converter logic
+* Enable CORS and review other deployment parameters
+* Integrate your Codec Server with the Temporal Web UI
+* Securely return decoded results in the CLI and the Web UI
+
+Make your changes to the code in the `practice` subdirectory (look for 
+`TODO` comments that will guide you to where you should make changes to 
+the code). If you need a hint or want to verify your changes, look at 
+the complete version in the `solution` subdirectory.
+
+
+## Part A: Configure a Codec Server to Use Your Data Converter
+
+1. First, you'll review a barebones Codec Server implementation in Go, and make
+   the necessary changes to integrate the custom data converter from Exercise 1.
+   Examine the `main.go` file in the `codec-server` subdirectory. This file
+   contains a complete HTTP server implementation using the Go standard library.
+   It listens on endpoints at `/{namespace}/encode` and `/{namespace}/decode` as
+   expected by the Temporal CLI, Web UI, and SDKs, and contains stubs for OIDC
+   and CORS enablement. These are the baseline requirements for a Temporal Codec
+   Server, which can be implemented using standard HTTP functionality in any
+   language of your choosing.
+2. Temporal Codec Servers need, at minimum, one additional configuration detail
+   before they can be deployed from sample code. Specifically, Codec Servers
+   need to import the Converter logic from your own application, and then map
+   the Converter logic on a per-Namespace basis. Edit the `import()` block at
+   the top of `codec-server/main.go` to import the rest of your application as a
+   Go module named `temporalconverters`. Don't forget that you can import Go
+   modules from GitHub URLs.
+3. Next, create an array named `codecs` in the `main()` function of
+   `codec-server/main.go`. Keys should be Namespace strings. Values should be of
+   the type `converter.PayloadCodec`. By default, you only need to assign the
+   `default` namespace to `{temporalconverters.NewPayloadCodec()}` from this
+   example.
+4. After making these additions, you should have a functioning Codec Server,
+   integrated with your application logic. Again, everything else in here is
+   configured as generically as possible — note that this example Codec Server
+   listens on port 8081, which is usually used in testing configurations — but
+   this fulfills all the requirements of a Temporal Codec Server, and you could
+   incorporate any other authentication requirements on top of HTTP as needed.
+   Run your Codec Server with `go run ./codec-server` from the root of your
+   project directory. This will block the terminal it runs in, and await
+   connections.
+5. Now you can retrieve the decoded output of your Workflow Execution from the
+   previous Exercise. From another terminal window, run `temporal workflow show \
```

Comment:
You have a backslash in this command, which is intended as a continuation character, but unnecessary since it's all on on line. It's a long command, so it would be better to split up such that each option is on its own line, but if that's not practical, then removing the unnecessary characters is the next best thing.

Reply:
Also, the command resulted in an error `Required flag "workflow-id" not set` (despite my version of the `temporal` command, 0.11.0, alleging that it supports `-w` as a synonym to `--workflow-id` command). When I replaced `-w` with `--workflow-id`, the command worked as expected.

Reply:
Good catch -- updated this to multiple lines with the full `--workflow-id` argument.